atoti-query-analyser/src/components/Summary/Summary.tsx

```diff
@@ -204,12 +206,32 @@ function MapView<K, V>({
 function QuerySummaryView({
   querySummary: summary,
   planInfo: info,
+  graph,
 }: {
   querySummary: QuerySummary;
   planInfo: PlanInfo;
+  graph: RetrievalGraph;
 }) {
+  const exportAsJson = () => {
+    const vertices = Array.from(graph.getVertices());
+    const edges = vertices
+      .flatMap((vertex) => Array.from(graph.getOutgoingEdges(vertex)))
+      .map((edge) => ({
+        begin: edge.getBegin().getUUID(),
+        end: edge.getEnd().getUUID(),
+        metadata: edge.getMetadata(),
+        uuid: edge.getUUID(),
+      }));
+
+    const blob = new Blob([JSON.stringify({ vertices, edges }, null, 2)], {
+      type: "application/json;charset=utf-8",
+    });
+    saveAs(blob, `ExecutionGraph ${info.mdxPass}.json`);
```

Comment:
Looks good but how do you use the exported data? It does not seem to be an option in the loader.

Reply:
After discussion, we agreed that it was for further analysis in python / R / etc.
So the PR is good as it is